is/esb/newmount.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 # $1 = Azure storage account name 
-# $2 = Azure storage account key 
-# $3 = Azure file share name 
-# $4 = Mountpoint path (required for multi-instance VMSS, else set to empty string) 
+# $2 = Azure storage account key # In the NFS file share,  key is not used 
```

Comment:
Do you need to keep this line? I was confused when I saw it because of the 2 x $2 parameters.

Reply:
sorry for the confusion. deleted it